base/MAPL_Generic.F90

```diff
@@ -1351,7 +1352,7 @@ recursive subroutine MAPL_GenericInitialize ( GC, IMPORT, EXPORT, CLOCK, RC )
    _VERIFY(STATUS)
    if (isPresent) then
       nra = ESMF_ConfigGetLen( STATE%CF, RC = STATUS)
-      _ASSERT( NRA > 0 .and. NRA < 10,'needs informative message')
+      _ASSERT( NRA > 0,'needs informative message')
```

Comment:
The "needs informative message" was generated during a brut force sweep that changed the macros.   Since you are changing the logic here, presumably you can come up with a better error message?

Reply:
It is hard to imagine what can cause this number to be non-negative - the counting is done by ESMF_ConfigGetLen (counts the number of entries for a specific keyword), so I am not sure what would be a "better" assert message. How about "This should never happen"